mod/ckeditor/views/default/js/ckeditor/boot.js

```diff
@@ -0,0 +1,16 @@
+define(function (require) {
```

Comment:
The convention this introduces automatically requires `$plugin_id/boot`. We could change the convention to something like `boot/$plugin_id`?